hx_lti_initializer/views.py

```diff
@@ -369,7 +371,8 @@ def course_admin_hub(request):
             'debug': debug,
             'starter_object': starter_object,
             'starter_object_id': object_id,
-            'starter_collection_id': collection_id
+            'starter_collection_id': collection_id,
+            'utm_source': request.session.session_key if not request.session['is_staff'] else '',
```

Comment:
Yeah, I think we merge and then put that in the backlog of technical debt type items to be addressed later.